### PR TITLE
Fix for mtu config with tap port test

### PR DESCRIPTION
**Pull request by dpathrabe**
Signed-off-by: dpathrabe <deepak.p.pathrabe@intel.com>

### CODE REVIEW

**Comment by dpathrabe** [comment]
[mtu_config_tap_fix_exec.log](https://github.com/ipdk-io/ovs/files/9633748/mtu_config_tap_fix_exec.log)
[mtu_config_tap_fix_build.log](https://github.com/ipdk-io/ovs/files/9633750/mtu_config_tap_fix_build.log)
Adding logs

**Comment by ipdksw** [comment]
Result:FAILURE JenkinsConsole:https://s3.ipdk.io/ovs/PR-71-1/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/30491bdf17f9964c1834e74d624cfa372d7d9b34

File: ptf_tests/tests/mtu_config_test_with_tap.py [hunk 1]
```diff
@@ -1,18 +1,3 @@
-# Copyright (c) 2022 Intel Corporation.
```
Comment (dkaranjai) [inline ptf_tests/tests/mtu_config_test_with_tap.py]: you need to put these back

**Review by dkaranjai — CHANGES_REQUESTED** [review]

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-71-2/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/79fde95ec7ddd949a1b58c2ebb22895b87d4f79f

**Review by dkaranjai — APPROVED** [review]

**Comment by ipdksw** [comment]
Result:ABORTED JenkinsConsole:https://s3.ipdk.io/ovs/PR-71-3/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/79fde95ec7ddd949a1b58c2ebb22895b87d4f79f

**Review by dkaranjai — APPROVED** [review]

**Review by sijubaker — APPROVED** [review]

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-71-4/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/568a47034ad5cbb33a41bcd9af822d9caf8bc733

**Pull request merged**